### PR TITLE
Add External args in build_examples.py

### PR DESCRIPTION
**Pull request by joonhaengHeo**
For using config argument in build_examples.py.

### CODE REVIEW

File: scripts/build/build_examples.py [hunk 1]
```diff
@@ -131,10 +131,14 @@ def ValidateTargetNames(context, parameter, values):
     help=(
         'Set pigweed command launcher. E.g.: "--pw-command-launcher=ccache" '
         'for using ccache when building examples.'))
+@click.option(
+    '--args',
+    default="",
+    help='Additional build argument')
```
Comment (arkq) [inline scripts/build/build_examples.py:137]: I'm not sure whether it's the right approach with such args, but if so, this should be documented better, e.g. like where these args are passed, what is the format, etc.?

File: scripts/build/builders/android.py [hunk 2]
```diff
@@ -340,6 +340,18 @@ def generate(self):
                 self.validate_build_environment()
 
             gn_args = {}
+
+            if self.args:
```
Comment (arkq) [inline scripts/build/builders/android.py:344]: Maybe it should be available for all platforms, not Android only?

**Review by andy31415 — CHANGES_REQUESTED** [review]
We have intentionally not allowed mixed arguments in build_examples, to force explicit instructions even when they end up as gn arguments.

What are the arguments that are considered here? Why not have separate variant entries for them?

**Comment by joonhaengHeo** [comment]
@andy31415 
I understand. Several features are being added on the Android platform, so it was uploaded during the review process. I will close the PR.

**Pull request closed without merging**